Neos.Media.Browser/Classes/Controller/AssetController.php

```diff
@@ -70,8 +70,11 @@
  */
 class AssetController extends ActionController
 {
-    use CreateContentContextTrait;
     use BackendUserTranslationTrait;
+    use BackendUserTranslationTrait {
+        BackendUserTranslationTrait::initializeObject as BackendUserTranslationTraitInitializeObject;
```

Comment:
I'm not very familiar with method aliasing, but shouldn't this be lower-cased if you use it like that in line 176?
```suggestion
        BackendUserTranslationTrait::initializeObject as backendUserTranslationTraitInitializeObject;
```

Reply:
@KommunikativCh @dlubitz ping

Reply:
I think we should align this. But in PHP methods and aliases are case-insensitve, so it works anyways.

Reply:
I quickly changed it. @bwaidelich

Reply:
Thanks. I would have changed it myself, but wasn't sure if that was done by intention